app/services/integrations/aggregator/invoices/payloads/netsuite.rb

```diff
@@ -88,6 +88,18 @@ def discounts
 
             output
           end
+
+          def limited_rate(precise_unit_amount)
+            unit_amount_str = precise_unit_amount.to_s
+
+            return precise_unit_amount if unit_amount_str.length <= 15
```

Comment:
Could you turn this magic number into a constant so that we know what it means ;)